notebooks/benchmark.py

```diff
@@ -0,0 +1,115 @@
+import os
+import simplejson as json
```

Comment:
Are there any special reasons to use `simplejson` over the stdlib `json`, besides the backward-compatibility in Pythn 2.6(-)? 

If not, I'd just use the standard `json`, or at use something like:
```
try: import simplejson as json
except ImportError: import json
``` here